### ISSUE TITLE
Document ssl renegotiation/add config property

### ISSUE THREAD

**Issue by floragunncom** [floragunncom/search-guard#368]
Document `-Djdk.tls.rejectClientInitiatedRenegotiation=true` and add an integration test.
Also check if it makes sense to introduce a new sg config prop for that.
For SG when can also think about disabling client initiated renego by default.

Refer to https://github.com/floragunncom/search-guard/issues/362

**Comment by floragunncom** [comment]
@jochenkressin maybe we should disable renego by default in SG 6?

**Comment by jochenkressin** [comment]
Yes, added it to SG6 roadmap. Feel free to close this issue, or close it with the arrival of SG6.
